docs/_posts/murat-gunay/2021-12-06-roberta_token_classifier_icelandic_ner_is.md

```diff
@@ -0,0 +1,135 @@
+---
+layout: model
+title: Icelandic NER Model
+author: John Snow Labs
+name: roberta_token_classifier_icelandic_ner
+date: 2021-12-06
+tags: [icelandic, roberta, token_classifier, ner, is, open_source]
+task: Named Entity Recognition
+language: is
+edition: Spark NLP 3.3.2
+spark_version: 2.4
+supported: true
+article_header:
+  type: cover
+use_language_switcher: "Python-Scala-Java"
+---
+
+## Description
+
+- This model is imported from `Hugging Face`.
+
+- It's been trained using `roberta` fine-tuned model on Icelandic texts.
```

Comment:
This model was fine-tuned on the MIM-GOLD-NER dataset for the Icelandic language, leveraging Roberta embeddings and `RobertaForTokenClassification`.